src/components/MoneyRequestConfirmationListFooter.tsx

```diff
@@ -618,7 +618,7 @@ function MoneyRequestConfirmationListFooter({
                     shouldShowRightIcon
                     title={iouAttendees?.map((item) => item?.displayName ?? item?.login).join(', ')}
                     description={`${translate('iou.attendees')} ${
-                        iouAttendees?.length && iouAttendees.length > 1 ? `\u00B7 ${formattedAmountPerAttendee} ${translate('common.perPerson')}` : ''
+                        iouAttendees?.length && iouAttendees.length > 1 && formattedAmountPerAttendee ? `\u00B7 ${formattedAmountPerAttendee} ${translate('common.perPerson')}` : ''
```

Comment:
Why we removed `\u00B7`? Was that by purpose? It caused https://github.com/Expensify/App/issues/61429

Reply:
Good question. I don't recall. @allroundexperts do you recall?